README.md

```diff
@@ -2,6 +2,10 @@
 
 *Minima is a one-size-fits-all Jekyll theme for writers*. It's Jekyll's default (and first) theme. It's what you get when you run `jekyll new`.
 
+***Disclaimer:** The information here may vary depending on the version you're using. Please refer to the `README.md` bundled
+within the theme-gem for information specific to your version. <br/>
```

Comment:
This can be also done by targeting a tag on GitHub: https://github.com/jekyll/minima/blob/v2.5.0/README.md